util/git/client.go

```diff
@@ -735,27 +735,33 @@ func (m *nativeGitClient) CommitSHA() (string, error) {
 	return strings.TrimSpace(out), nil
 }
 
-// returns the meta-data for the commit
+// RevisionMetadata returns the metadata for the commit
 func (m *nativeGitClient) RevisionMetadata(revision string) (*RevisionMetadata, error) {
-	out, err := m.runCmd("show", "-s", "--format=%an <%ae>|%at|%B", revision)
+	author, err := m.runCmd("show", "-s", "--format=%an", revision)
```

Comment:
Have you considered using `%n` (newline) to separate tokens in the string instead of running multiple commands, i.e. `%an <%ae>%n%at%n%B`? I'd believe that a newline isn't allowed in email or name.

Reply:
I did actually - #15952 had that implementation. I was concerned that there was a potential for a newline in the name, but now that I looked closer that doesn't seem to be the case. That's what I get for doing this late at night...

Reply:
I set it back to the original impl from @dosmanak, PTAL

Reply:
I just tested it and git binary really strips the newline inside the author name.
```
git commit --author=$'first\nsecond <dosmanak@test>' -am "resetme"
git show -s --format="%an%x00%at"| xxd
00000000: 6669 7273 7473 6563 6f6e 6400 3137 3236  firstsecond.1726
00000010: 3932 3532 3737 0a                        925277.
```

Reply:
I just tested it and git binary really strips the newline inside the author name.
```
$ git commit --author=$'first\nsecond <dosmanak@test>' -am "resetme"
$ git show -s --format="%an%x00%at"| xxd
00000000: 6669 7273 7473 6563 6f6e 6400 3137 3236  firstsecond.1726
00000010: 3932 3532 3737 0a                        925277.
```